modules/Bio/EnsEMBL/ArchiveStableId.pm

```diff
@@ -103,6 +105,8 @@ sub new {
   $self->{'release'} = $release;
   $self->{'assembly'} = $assembly;
   $self->{'type'} = $type;
+# species_id member attribute added to support multiSpecies species_id tests on archiveStable.t
+  $self->{species_id} = ""; 
```

Comment:
species_id cannot be set at construction? Is it auto-set when the object gets stored?